README.md

```diff
@@ -1 +1,56 @@
-# mailserver
\ No newline at end of file
+# mailserver
+
+This server is supposed to handle TCP connections to receive emails from clients.
+The vision is a fully functional mailserver to manage your emails.
+
+
+## Functionality
+- [ ] Receive Emails
+  - [x] Parse incoming Emails .
+  - [ ] Store Emails in a Mailbox.
+    - [ ] Create a Mailbox with multiple Emails.
+  - [ ] Store Emails in a Folder.
+  - [ ] Send Emails to the cc,bcc, to-Addresses.
+- [ ] Synchronization
+  - [ ] provide a list of emails when a client connects.
+  - [ ] provide a list of emails when a client requests a refresh.
+  - [ ] provide a list of emails with according folders instead.
+- [ ] Filtering
+  - [ ] Filter incoming messages Spam to a spam folder
+
+## Domains
+
+## Ubiquitous Language
+- `Recieving Email` : Emails that are sent to this server. This can happen by a client or in the future by another mailserver
```

Comment:
Oh, okay. Now the definition above is now clear for me.  But maybe the definition is a bit too late or we change the definition of our mail server and describe it a bit more generally. line 3-4

Reply:
I added IMAP and SMTP to the server definition